addons/udes_stock/models/stock_location.py

```diff
@@ -106,9 +104,51 @@ def _domain_height_category(self):
 
                 If not set then the location will use the format
                 of the nearest ancestor that has a format specified.
-                """
+                """,
+    )
+    u_state = fields.Selection(
```

Comment:
Question: I know that fields that we add have the underscore, maybe in this case we may need an exception.
Field state,active, parent_id example are special fields which i think is better to be state as can be easier on xml to add invisible attributes

Reply:
This is not meant as a standard state field, it doesn't track progress or anything like that.

Reply:
Could be worth renaming it something like `u_countable_state` instead, to be a bit more explicit that it doesn't apply to all locations.